### PR TITLE
refactor: backend settings v2

### PR DESCRIPTION
**Pull request by kwajiehao**
This PR is modifies frontend components which hit the backend GET `/{siteName}/settings` endpoint in response to changes to the settings response format made in the backend PR #[282](https://github.com/isomerpages/isomercms-backend/pull/282). 

The biggest change to note is the removal of the `footerSha` state variable in the `EditContactUs` layout. The GET `/{siteName}/settings`  endpoint pulls the existing footer file to check for differences in the request footer fields and the current repo value - in doing so, it retrieves the existing footer file sha. Thus, we no longer need to submit a sha value when updating the footer. 

Additional refactoring to bring the settings component in line with our existing refactor is in a future PR : https://github.com/isomerpages/isomercms-frontend/pull/666

### CODE REVIEW

File: src/layouts/EditContactUs.jsx [hunk 1]
```diff
@@ -202,12 +200,8 @@ const EditContactUs = ({ match }) => {
         const settingsResp = await axios.get(
           `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/settings`
         )
```
Comment (alexanderleegs) [inline src/layouts/EditContactUs.jsx:202]: Also using the v1 endpoint

File: src/utils/siteColorUtils.js [hunk 2]
```diff
@@ -134,10 +134,9 @@ const getSiteColors = async (siteName) => {
     const settingsResp = await axios.get(
       `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/settings`
     )
```
Comment (alexanderleegs) [inline src/utils/siteColorUtils.js:136]: Also using the v1 endpoint

**Review by alexanderleegs — CHANGES_REQUESTED** [review]
PR looks okay - but we should swap out the v1 endpoints for settings for the v2 endpoints. Apart from the commented places, we also need to swap them out in lines 118 and 356 in the Settings layout file

File: src/utils/siteColorUtils.js (hunk 2 above)
Comment (kwajiehao) [reply src/utils/siteColorUtils.js]: > PR looks okay - but we should swap out the v1 endpoints for settings for the v2 endpoints. Apart from the commented places, we also need to swap them out in lines 118 and 356 in the Settings layout file

Fixed with [4117e66](https://github.com/isomerpages/isomercms-frontend/pull/618/commits/4117e66f1834aaeaa427d0168a9d235a3f30c589)

**Comment by kwajiehao** [comment]
> PR looks okay - but we should swap out the v1 endpoints for settings for the v2 endpoints. Apart from the commented places, we also need to swap them out in lines 118 and 356 in the Settings layout file

Fixed with [4117e66](https://github.com/isomerpages/isomercms-frontend/pull/618/commits/4117e66f1834aaeaa427d0168a9d235a3f30c589)

**Comment by alexanderleegs** [comment]
Superceded by https://github.com/isomerpages/isomercms-frontend/pull/666

**Pull request closed without merging**